nautobot/docs/administration/nautobot-shell.md

```diff
@@ -57,14 +57,14 @@ Use a `for` loop to cycle through all objects in the list:
 
 To count all objects matching the query, replace `all()` with `count()`:
 
-```
+```python
 >>> Device.objects.count()
 1274
 ```
 
 To retrieve a particular object (typically by its primary key or other unique field), use `get()`:
 
-```
+```python
 >>> Site.objects.get(pk=7)
```

Comment:
We might want to update this example since we updated the pk to be a UUID.

Reply:
Oh yes! I will do that.